content/en/docs/metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Build Controller Metrics
```

Comment:
Note that this is geared towards installation/configuration of the controller. More is coming in this regard as we are tuning our installation process.